dashi/rnaseqc/over_time.py

```diff
@@ -2,11 +2,13 @@
 import dash_core_components as dcc
 import dash_html_components as html
 import dash.dependencies as dep
+import dash.exceptions
+import itertools
 
 import plotly
+import sd_material_ui
```

Comment:
I got a `ModuleNotFoundError: No module named 'sd_material_ui'` error, does this need to be added to `setup.py`?

Reply:
yes, it can be installed with `pip install sd_material_ui`

Reply:
It does need to be added to `setup.py`. I will do so.

Reply:
@callunity I have added it to the `setup.py` file.